test/integration/kubernetes_deploy_test.rb

```diff
@@ -686,4 +686,16 @@ def test_ejson_secrets_respects_no_prune_flag
     ejson_cloud.assert_secret_present('catphotoscom', type: 'kubernetes.io/tls', managed: true)
     ejson_cloud.assert_secret_present('ejson-keys', managed: false)
   end
+
+  def test_invalid_context
+    old_config = ENV['KUBECONFIG']
+    begin
+      ENV['KUBECONFIG'] = File.join(__dir__, '../fixtures/invalid_config.yml')
```

Comment:
Wouldn't the path this generates have literal `..` in it? Does that actually work?

Reply:
It does work. I can see if it continues to work without the `..`